server/testdriver/core/src/main/java/org/infinispan/server/test/core/ForkedInfinispanServerDriver.java

```diff
@@ -40,16 +42,26 @@
    private static final int SHUTDOWN_TIMEOUT_SECONDS = 15;
    private final List<ForkedServer> forkedServers = new ArrayList<>();
    private final List<Path> serverHomes;
+   private final String serverDataPath;
 
    protected ForkedInfinispanServerDriver(InfinispanServerTestConfiguration configuration) {
       super(configuration, InetAddress.getLoopbackAddress());
       String globalServerHome = configuration.properties().getProperty(TestSystemPropertyNames.INFINISPAN_TEST_SERVER_DIR);
       if (globalServerHome == null || globalServerHome.isEmpty()) {
          throw new IllegalArgumentException("You must specify a " + TestSystemPropertyNames.INFINISPAN_TEST_SERVER_DIR + " property.");
       }
+      serverDataPath = System.getProperty(Server.INFINISPAN_SERVER_ROOT_PATH);
+      if (serverDataPath != null && !serverDataPath.trim().isEmpty() && !serverDataPath.contains("%d")) {
+         throw new IllegalStateException("Server root path should have the index. Add the %d regex to the path. Example: /path/to/server_%d");
+      }
       this.serverHomes = new ArrayList<>();
       Path src = Paths.get(globalServerHome).normalize();
-      for (int i = 0; i < configuration.numServers(); i++) {
+      // ISPN-13109
```

Comment:
I think we should either remove this comment, or add a little description to go with the Jira ID.

Reply:
Removed